src/UI/SignTransactionsModals/SignWithDeviceModal/SignStep.tsx

```diff
@@ -106,33 +108,77 @@ export const SignStep = ({
   const showProgressSteps = allTransactions.length > 1;
 
   const classes = getGeneratedClasses(className, true, {
-    formGroup: 'form-group text-left',
-    formLabel: 'form-label text-secondary',
-    icon: 'text-white',
-    contentWrapper:
-      'd-flex flex-column justify-content-start flex-md-row justify-content-md-between mb-3',
-    tokenWrapper: 'mb-3 mb-md-0 d-flex flex-column align-items-start',
-    tokenLabel: 'text-secondary text-left',
-    tokenValue: 'd-flex align-items-center mt-1',
-    scamReport: 'text-warning',
-    scamReportIcon: 'text-warning mr-1',
-    tokenAmountLabel: 'text-secondary text-left',
-    tokenAmountValue: 'd-flex align-items-center',
-    dataFormGroup: 'form-group text-left',
-    errorMessage:
-      'text-danger d-flex justify-content-center align-items-center',
-    buttonsWrapper: 'd-flex align-items-center justify-content-end mt-spacer',
-    cancelButton: 'btn btn-dark text-white flex-even mr-2',
-    signButton: `btn ${
-      scamReport ? 'btn-warning' : 'btn-primary'
-    } flex-even ml-2`
+    formGroup: classNames(
```

Comment:
These should probably be moved to a separate file, it's a bit of repetitive code that we don't really need to debugging the behaviour of the component